backend/server/src/db/mod.rs

```diff
@@ -0,0 +1,33 @@
+mod sqlite;
+
+use once_cell::sync::OnceCell;
+use sqlx::sqlite::{SqliteConnectOptions, SqliteJournalMode, SqlitePool};
+use tracing::debug;
+
+static SQLITEPOOL: OnceCell<SqlitePool> = OnceCell::new();
```

Comment:
not sure if this is the best design.

Eventually I would like for the consuming code to look like:
```
sql_facade::insert::build_status(build_status_struct);
```
How the connection pool is maintained, I would like to not have the calling code know about it.

Reply:
Sounds good

Reply:
going to punt on this for now. I agree we should do that, but I would like to get the sqlx in now, and it will be easier to do small iterations.